scp/scpHandlingEventThatInitiatesAgentSCPProgram.cpp

```diff
@@ -50,20 +50,13 @@ ScResult ASCPHandlingEventThatInitiatesAgentSCPProgram::DoProgram(ScElementaryEv
     else
     {
       SC_AGENT_LOG_WARNING(
-          "Max customer waiting time"
+          "Max customer waiting time "
           << maxCustomerWaitingTime
-          << " has expired before action of class `action_scp_interpretation_request` was initiated because"
+          << " has expired before action of class `action_scp_interpretation_request` was initiated because "
           << timeFromStart << " ms have passed");
       scpAction.InitiateAndWait();
     }
   }
-  auto const & resIt = m_context.CreateIterator5(
-      action, ScType::ConstCommonArc, ScType::ConstNode, ScType::ConstPermPosArc, ScKeynodes::nrel_result);
-  if (resIt->Next())
-  {
-    ScAddr const & actionResult = resIt->Get(2);
-    action.SetResult(actionResult);
-  }
   return action.FinishSuccessfully();
```

Comment:
something similar should be done with action.FinishSuccessfully
sometimes actions are finished successfully 2 times
also it is possible that scp-program finishes action unsuccessfully and then this agent generates arc from action_finished_successfully